extractor/src/main/java/org/schabi/newpipe/extractor/services/soundcloud/linkHandler/SoundcloudStreamLinkHandlerFactory.java

```diff
@@ -30,6 +30,7 @@ public String getUrl(String id) throws ParsingException {
     @Override
     public String getId(String url) throws ParsingException {
         Utils.checkUrl(URL_PATTERN, url);
+        if (url.charAt(url.length() -1) == '/') url = url.substring(0, url.length()-1);
```

Comment:
Please add a comment that the trailing backslash needs to be removed due to inconsistency in the SoundCloud API

Reply:
resolved in 6a70cb9d50132c1cf1bf59478f029b199d65fefc